rpmconf/rpmconf.py

```diff
@@ -340,13 +340,13 @@ def _clean_orphan_file(rpmnew_rpmsave):
         # rpmnew_rpmsave is lowercase name of rpmnew/rpmsave file
         (rpmnew_rpmsave_orig, _) = os.path.splitext(rpmnew_rpmsave)
         package_merge = file_delete = None
-        try:
-            package_merge = subprocess.check_output(
-                ["/usr/bin/rpm", '-qf',
-                 rpmnew_rpmsave_orig, '--qf', '%{name}'],
-                universal_newlines=True, stderr=subprocess.DEVNULL)
-        except subprocess.CalledProcessError:
+        trans = rpm.TransactionSet()
+        # pylint: disable=no-member
+        tmp_db = trans.dbMatch("basenames", rpmnew_rpmsave_orig)
+        if len(tmp_db) == 0:
             file_delete = rpmnew_rpmsave
+        else:
+            package_merge = tmp_db[0].Name
```

Comment:
This line gives me (in python3):
  TypeError: 'rpm.mi' object is not subscriptable